src/types/hr/hr.type.ts

```diff
@@ -0,0 +1,10 @@
+export type Hr = {
+  id: string;
+  email: string;
+  fullName: string;
+  password: string;
+  role: string;
+  company: string;
+  max_reserved_students: number;
+  refresh_token: string;
```

Comment:
A tu to już totalny miszmasz

Reply:
Jakbyś mógł rozwinąć było by spoko, bo nie bardzo rozumiem🤔

Reply:
no masz w jednym typie dwa różne kejsy